cv32e40x/tests/programs/custom/illegal_instr_test/illegal_instr_test.S

```diff
@@ -47699,6 +47603,79 @@ main:
     nop
 .word(0x040080B3) # p.abs   [31:25]==000_0010, [14:12]==000, [6:0]==011_0011
 
+    jal check_gpr_integrity
```

Comment:
This code was previously inlined, checks that the state of the gprs have not been changed due to executing instructions that are supposedly invalid